discord/user.py

```diff
@@ -175,6 +179,18 @@ def display_avatar(self) -> Asset:
         """
         return self.avatar or self.default_avatar
 
+    @property
```

Comment:
this should be cached.

Reply:
Why?

Reply:
On thinking again you're right, my mistake.
Brain fart.

Reply:
👍